source/application/BioXAS/BioXASImagingAppController.cpp

```diff
@@ -112,7 +91,7 @@ void BioXASImagingAppController::setupExporterOptions()
 	if (matchIDs.count() != 0)
 			bioXASDefaultXAS->loadFromDb(AMDatabase::database("user"), matchIDs.at(0));
 
-	bioXASDefaultXAS->setName("IDEAS Default XAS");
+	bioXASDefaultXAS->setName("BioXASImaging Default XAS");
```

Comment:
it happens when we copy/paste codes. There is a similar one in PGMAppController.cpp.